.travis.yml

```diff
@@ -6,12 +6,18 @@ android:
     - tools
     - build-tools-28.0.3
     - android-28
+    - android-30
+    - sys-img-x86-android-27
```

Comment:
Since most android devices are based on arm processors, please use `sys-img-armeabi-v7a-android-<your emulator level>`

Reply:
Since my emulator level, the one present currently in .travis.yml, and the one suggested in the comment linked to issue #168 - [131](https://github.com/anitab-org/mentorship-android/pull/131#issuecomment-459584221) is x86 I used that. Please let me know if I still have to change it to armeabi-v7a.

Reply:
You should change it as the x86 abi image is invalid  `Error: Invalid --abi x86 for the selected target` is occurring, you can checkout this from [here](https://travis-ci.org/github/anitab-org/mentorship-android/builds/682240001)

Reply:
I had seen the error but before that when I ran with x86 the error was a kernel qemu file missing following which I read an answer on Travis Build working only till 27, that's why I didn't change it. I will change to to `armeabi-v7a`.

Reply:
`$ echo no | android create avd --force -n test -t android-27 --abi armeabi-v7a
Valid ABIs: no ABIs.
Error: Invalid --abi armeabi-v7a for the selected target.
The command "echo no | android create avd --force -n test -t android-27 --abi armeabi-v7a" failed and exited with 1 during .`
The emulator whose ini file I renamed to test.ini has the following specs.
![android27pixelspecs](https://user-images.githubusercontent.com/48416306/80865724-4b0ab080-8c9c-11ea-9785-a9b7a026c0da.png)